### ISSUE TITLE
Improve accessibility labelling

### ISSUE THREAD

**Issue by oherst** [Giphy/giphy-react-native-sdk#113]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

All buttons and images (or gifs) should have appropriate accessibility labelling.

### Motivation 🔦

Hello! I believe this to be an enhancement rather than a bug so I figured a feature request would be more appropriate but I can make this request somewhere else if this is not the right place.

We are using the Giphy RN sdk in our app and have recently been audited by a third party for accessibility. They picked up some issues with our Giphy implementation and I wondered if you might be able to give your thoughts on them?

- Trending buttons don't have a name (they just announce "button").
- When previewing a gif the back button and "view on giphy button" don't have names.
- When previewing a gif the gif cannot be focused.

Additionally I note that the buttons at the bottom of the giphy main window are read as "sticker underscore type underscore button, button" which could be better (eg, sticker button). To be clear, this was not picked up by the auditors it is just something I noticed whilst testing.

Would this be something you would consider for a future enhancement?

Thanks very much.

![Screenshot 2023-06-05 at 14 44 18](https://github.com/Giphy/giphy-react-native-sdk/assets/2856248/3755f644-f817-4da2-b890-b07974da06fd)
![Screenshot 2023-06-05 at 14 44 13](https://github.com/Giphy/giphy-react-native-sdk/assets/2856248/48b9bdbe-3351-4a19-a11f-9e6621d8988f)
![Screenshot 2023-06-05 at 14 44 47](https://github.com/Giphy/giphy-react-native-sdk/assets/2856248/b88d2978-a0be-41c6-9eed-60cbe5f6e1e3)
![IMG_2C41A574B9CA-1](https://github.com/Giphy/giphy-react-native-sdk/assets/2856248/df2e3bc6-281b-460f-9c28-186f83276c85)

**Comment by pshoniuk** [comment]
Hi @oherst
Thanks for the detailed report. I’ll pass this feedback on to my team & keep you in the loop.

**Comment by efstathiosntonas** [comment]
thank you for this guys

**Comment by oherst** [comment]
thanks so much, can't wait to try it out!
